### ISSUE TITLE
How to change Torque Target in Torque Mode to achieve electronic braking?

### ISSUE THREAD

**Issue by fender90** [EmanuelFeru/hoverboard-firmware-hack-FOC#81]
Hi Emanuel.
First let me congratulate you on your fantastic job!
I am making a robotic platform which should be controlled via Arduino. At the moment I have connected a simple RC receiver to it but I plan to implement a more sophisticated ground control unit later. 
I have selected torque mode in the firmware and would like to have something like electronic braking.
The documentation says the following.
_TORQUE MODE: in this mode the input torque target is realized. This mode enables motor "freewheeling" when the torque target is 0. Recommended for most applications with a sitting human driver. If motor braking is desired instead of "freewheel" when torque target is 0, then a torque target below 0 should be set when speedAvgAbs > 0._

Could you please specify on how to do that? I didn't find where I can change the torque target value. Also is it possible to have some kind of "stationary brake" option? Maybe when speedAvgAbs >= 0._ ?

**Comment by juodumas** [comment]
@EmanuelFeru looks like this question comes a lot and everyone seems to be reading the README. So I would like to suggest to extend the torque mode description and clarify its differences/advantages/disadvantages to other modes...

**Comment by fender90** [comment]
That would be perfect. Thank you very much.

**Comment by EmanuelFeru** [comment]
That can be implemented, if i have time I will do it over the weekend.

**Comment by EmanuelFeru** [comment]
@fender90 question, why have you decided to use Torque mode and not Voltage or Speed for your robot?
Because Voltage or Speed will stop when input is 0.

**Comment by fender90** [comment]
> @fender90 question, why have you decided to use Torque mode and not Voltage or Speed for your robot?
> Because Voltage or Speed will stop when input is 0.

Because the platform I am making is going to carry heavy load.

**Comment by EmanuelFeru** [comment]
With Voltage or Speed you can carry the same load. The control mode does not influence the maximum load it can carry.

**Comment by fender90** [comment]
> With Voltage or Speed you can carry the same load. The control mode does not influence the maximum load it can carry.

O_o Thanks for the clarification, but then it is not clear what is the purpose of other modes and what are the advantages/disadvantages of each one.

**Comment by EmanuelFeru** [comment]
@fender90, @juodumas  In my last commit f2d86f3b309b722ae705d0149b7557732cc0c202 I added Electric Brake and Standstill hold functionality. Also I added a table in Readme to help you choose the methods that suits your application and needs.
![image](https://user-images.githubusercontent.com/24780745/87871722-27115e00-c9b3-11ea-83fc-c0ebdca3ab2e.png)

Feel free try and choose the behavior that you like or need.
